app/views/govuk_publishing_components/component_guide/component_doc/_call.html.erb

```diff
@@ -1,12 +1,20 @@
+<%
+component_code =
+  if example.has_block?
+    "<%= render \"#{@component_doc.partial_path}\", #{example.pretty_data} do %\>\n" +
+    "  #{example.block.html_safe}" +
+    "<% end %\>"
+  else
+    "<%= render \"#{@component_doc.partial_path}\", #{example.pretty_data} %\>"
+  end
+
+code =
+  if example.has_embed?
+    example.embed.sub('<%= component %' + '>', component_code)
```

Comment:
Why do we have a '+' here?

Reply:
Having `%>` together terminates the embedded ruby block. I couldn't find a better way to escape. Do you know one?

I'll update commit message to mention it.

Reply:
Oh darn there's one in the lines above 😞. Massive d'oh!

Reply:
That looks better :-).